### PR TITLE
added necessary components

### PR DESCRIPTION
**Pull request by sakshambhatt**
Please review my component library and provide feedback.
Thanks!

### CODE REVIEW

**Comment by ishanjirety** [comment]
image

**You should never use uppercase letters to create classes.**
Other things look good to me

**Comment by AnkitKarnAK** [comment]
Hey Saksham
Add `cursor: pointer` property on primary button

**Comment by sakshambhatt** [comment]
Hello, I have changed all the class names to lowercases. Thanks for reviewing @ishanjirety .
I tried to use `cursor: pointer` in button, but it did not work. I referred [this stackoverflow answer](https://stackoverflow.com/questions/18434626/why-is-cursorpointer-effect-in-css-not-working/44510398), and yet I could not do it @AnkitKarnAK

**Comment by sakshambhatt** [comment]
Hi @AnkitKarnAK , `cursor: pointer` is working now. I have not changed anything. I don't know what happened, but it is working now. Thanks for reviewing.

File: css/prism.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
+/* PrismJS 1.23.0
+https://prismjs.com/download.html#themes=prism-okaidia&languages=markup+css+clike+javascript */
+/**
+ * okaidia theme for JavaScript, CSS and HTML
+ * Loosely based on Monokai textmate theme by http://www.monokai.nl/
+ * @author ocodia
+ */
```
Comment (saisumanthkacham) [inline css/prism.css]: Remove the unwanted code here and no suggestions in code review, everything looks good and descriptive and I love your codebase its clean.

File: css/prism.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+code[class*="language-"],
+pre[class*="language-"] {
+	color: #f8f8f2;
+	background: none;
+	text-shadow: 0 1px rgba(0, 0, 0, 0.3);
+	font-family: Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
+	font-size: 1em;
```
Comment (saisumanthkacham) [inline css/prism.css:7]: Ignore the selected lines as I am writing here Ui review
1.Your implementation of the minimal design can be appreciated here @sakshambhatt 
2.the hover state of links on the sidebar and navbar should be changed (remove text-decoration and change the hover state colour)
3.The position of the sidebar should be fixed to improve the user experience.
4.The place at the logo is junky, change the things around it.

**Review by saisumanthkacham — CHANGES_REQUESTED** [review]

File: css/prism.css (hunk 2 above)
Comment (sakshambhatt) [reply css/prism.css:7]: Thank you for your kind review. I have finally gotten around to actually take your advice. Please take a look @saisumanthkacham.

**Pull request merged**